crates/sargon/src/profile/mfa/security_structures/matrices/builder/matrix_builder.rs

```diff
@@ -164,6 +164,14 @@ impl MatrixBuilder {
         Ok(())
     }
 
+    pub fn validate_primary_role_in_isolation(
```

Comment:
@danvleju-rdx do this in a PR, but for each role

Reply:
you need to put it in `sargon-uniffi` crate too:

```rust
#[uniffi::export]
impl SecurityShieldBuilder {
   pub fn validate_primary_role_in_isolation(&self) ...
   pub fn validate_recovery_role_in_isolatoin(&self) ...
}
```

or maybe you want:

```rust
#[uniffi::export]
impl SecurityShieldBuilder {
   pub fn validate_role_in_isolation(&self, role: RoleKind) {
        self.get(|builder| {
              match role {
                 RoleKind::Primary => builder.validate_primary_role_in_isolation
             }
       })
   }
}
```
.......